db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2019_04_15_121556) do
+ActiveRecord::Schema.define(version: 2019_05_16_213012) do
```

Comment:
This slipped in @pezholio.

Reply:
Ugh. That'll teach me to blindly commit everything